### ISSUE TITLE
First page refresh causes wrong image positions

### ISSUE THREAD

**Issue by bartoszmalec** [miromannino/Justified-Gallery#150]
I have a problem with grid on first page load. After first page load or hard refresh (ctrl + F5), grid looks bad, and images has incorrectly calculated widths (width of images is 100%). After normal page refresh everything back to narmal and images has good positions and sizes.

Have you ever met this problem?

**Comment by bartoszmalec** [comment]
![test1](https://cloud.githubusercontent.com/assets/13161733/13204698/d38fdee0-d8d6-11e5-9376-b2a58f23d8ef.png)
![test2](https://cloud.githubusercontent.com/assets/13161733/13204699/d415d568-d8d6-11e5-9b95-93b4476a4009.png)

**Comment by miromannino** [comment]
Yes it happened one time... can you send me a link to this page? Or with the mail the html with the images (saving the page with the browser)?
Thank you

**Comment by bartoszmalec** [comment]
I sent an email to you with access credentials.

**Comment by bartoszmalec** [comment]
I resolved this problem. I run justifiedGalleryCode in 
`$("document").ready(function() {
    $("document").load(function() {
        [justifiedGalleryCode]
    });
});`

**Comment by Sebastiansc** [comment]
@bartosz-malec what is that justifiedGalleryCode?

**Comment by voletiv** [comment]
@bartosz-malec  I'm facing a similar issue where the widths are 100% in all cases, could you please tell us what your JustifiedGalleryCode is?

**Comment by danygit** [comment]
> var isDomReady = false;
var isWinLoad = false;

$(document).ready(function() {
    isDomReady = true;
    initJustifiedGallery();
});

$(window).load(function() {
    isWinLoad = true;
    initJustifiedGallery();
});

function initJustifiedGallery()
{
   $('.gallery').justifiedGallery({
                rowHeight: 180,
                lastRow: 'nojustify',
                margins: 6,
                selector: 'div'
            });
}
